access-control-app/src/cmr/access_control/api/routes.clj

```diff
@@ -2,6 +2,7 @@
   "Defines the HTTP URL routes for the access-control API."
   (:require
    [cheshire.core :as json]
+   [clojure.walk :as w]
```

Comment:
w is to short I thought, what about "walk"

Reply:
Good point! Changed to `walk`